Update caption.md (#2972)
enkidevs/curriculum: web/html/tables/caption.md

</table>
```

> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 

> If you'd like to only style the elements bold without bringing attention to them, you should replace it with the `style="font-weight:bold;"` CSS property.

Reviewer: What is "it" in this case?

Author: the `<b>` element.

These two are connected: 

> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 

If you'd like to only style the elements bold without bringing attention to them, you should replace it with the `style="font-weight:bold;"` CSS property.

Should I add it as a single note?

Author: I've added a span and modified the wording, lmk if it's good

</table>
```

In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 

> If you'd like to only style the elements bold without bringing attention to them, you should replace it with a `<span>` element, and apply the `style="font-weight:bold;"` CSS property to it:

Reviewer: ```suggestion
If you'd like to only style the elements bold without bringing attention to them, you can use a `<span>` element, and apply the `style="font-weight:bold;"` CSS property to it:
```
